front/lib/resources/group_resource.ts

```diff
@@ -652,6 +652,22 @@ export class GroupResource extends BaseResource<GroupModel> {
     workspace: LightWorkspaceType,
     transaction?: Transaction
   ) {
+    const groups = await this.model.findAll({
+      attributes: ["id"],
+      where: { workspaceId: workspace.id },
+      transaction,
+    });
+
+    const groupIds = groups.map((group) => group.id);
+
+    await GroupVaultModel.destroy({
+      where: {
+        groupId: {
+          [Op.in]: groupIds,
+        },
+      },
+      transaction,
+    });
     await GroupMembershipModel.destroy({
       where: {
         workspaceId: workspace.id,
```

Comment:
maybe we could reuse the groupIds here and below to be consistent with the new destroy   ? ( `groupId: groupIds` )